docs/lib/restapi/fragments/flutter/getting-started/40_postTodo.md

```diff
@@ -0,0 +1,15 @@
+```dart
+try{
+    RestOptions options = RestOptions(
+        path: "/todo"
+        body: Uint8List.fromList("{\"name\":\"Mow the lawn\"}".codeUnits)
```

Comment:
What's the Uint8List?

Reply:
It's a way to store 8-bit unsigned integers.  With the Flutter Method Channel this is how the equivalent Data stored in Android and iOS is converted into.  

https://api.dart.dev/stable/2.10.4/dart-typed_data/Uint8List-class.html